test/Base_Tests/src/System/Embedded_Native_Libs_Spec.enso

```diff
@@ -0,0 +1,127 @@
+from Standard.Base import all
+import Standard.Base.Runtime.Ref.Ref
+from Standard.Test import all
+
+polyglot java import java.util.jar.JarFile
+polyglot java import java.util.jar.JarEntry
+polyglot java import java.nio.file.Path as JPath
+polyglot java import java.util.Enumeration
+
+only_on_linux = if System.os == "linux" then Nothing else "Test runs only on Linux"
```

Comment:
- why do we execute the test only on Linux?
- I believe we want to avoid the libraries on other systems as well
- the test seems generic enough to run on other OSes

Reply:
The only reason was because of explicit path separator in https://github.com/enso-org/enso/blob/12e1970a9e282959ec826e9dc0f4b2a949e46def/test/Base_Tests/src/System/Embedded_Native_Libs_Spec.enso#L45. But you are right, this test **should** run on all platforms. I will create a fix soon.

Reply:
Fix is on #13174